soc/arm/st_stm32/stm32f4/Kconfig.series

```diff
@@ -13,5 +13,15 @@ config SOC_SERIES_STM32F4X
 	select HAS_STM32CUBE
 	select CPU_HAS_ARM_MPU
 	select HAS_SWO
+	select HAS_PM
 	help
 	  Enable support for STM32F4 MCU series
+
+if SOC_SERIES_STM32F4X
```

Comment:
This shouldn't be required

Reply:
Done.

Reply:
Unfortunately, it is required to avoid some dependency loops.

Reply:
In that case, move to Kconfig.defconfig.series file, which I guess is made for this

Reply:
Done.